src/Microsoft.ML.FastTree/Training/EnsembleCompression/LassoBasedEnsembleCompressor.cs

```diff
@@ -533,7 +533,7 @@ private unsafe void LoadTargets(double[] trainScores, int bestIteration)
             }
         }
 
-        public bool Compress(IChannel ch, TreeEnsemble ensemble, double[] trainScores, int bestIteration, int maxTreesAfterCompression)
+        bool IEnsembleCompressor<short>.Compress(IChannel ch, TreeEnsemble ensemble, double[] trainScores, int bestIteration, int maxTreesAfterCompression)
```

Comment:
>bool [](start = 8, length = 4)

add explicit private access modifier everywhere.

Reply:
This will not work for internal interface.

---
In reply to: [251971388](https://github.com/dotnet/machinelearning/pull/2243#discussion_r251971388) [](ancestors = 251971388)